### PR TITLE
Pull `cypress` image

### PR DESCRIPTION
**Pull request by peterjan**
# PULL REQUEST

## Overview

This PR extends the `portal-role-task.yml` file with an extra step that pulls the `cypress` image. This avoids having to pull it on request when the abuse scanner needs it, ensuring a smooth parse of the abuse email rather than hanging for x seconds waiting on the image being pulled.

It also adds support for the `ABUSE_TMP_DIR` env. variable.

## Example for Visual Changes

N/A

## Checklist

- [x] All git commits are signed. (REQUIRED)
- [x] All new methods or updated methods have clear docstrings.
- [ ] Testing added or updated for new methods.
- [ ] Verified if any changes impact the WebPortal Health Checks.
- [ ] Appropriate documentation updated.
- [ ] Changelog file created.

## Issues Closed

Closes SKY-730

### CODE REVIEW

File: playbooks/templates/.env.j2 [hunk 1]
```diff
@@ -293,6 +293,11 @@ ABUSE_PORTAL_URL={{ webportal_common_config.abuse_portal_url }}
 # REQUIRED: Hardcoded, not configurable
 ABUSE_SPONSOR="Abuse Scanner"
 
+# Temporary directory used by the abuse scanner
+#
+# OPTIONAL: manually defined by user
+ABUSE_TMP_DIR={{ webportal_common_config.abuse_tmp_dir | default('/tmp/abuse-scanner') }}
```
Comment (MSevey) [inline playbooks/templates/.env.j2:299]: What is this used for?

A tmp directory doesn't feel like a needed environment variable.  If it is just used during setup then we can just have an ansible variable.

File: playbooks/tasks/portal-role-task.yml [hunk 2]
```diff
@@ -365,3 +365,10 @@
       command: crontab -u {{ webportal_user }} {{ webportal_cron_file }}
 
   when: webportal_setup_health_checks
+
+# Pull cypress image to avoid lazily pulling it on the fly when the abuse
+# scanner needs it
+- name: Pull cypress docker image
```
Comment (MSevey) [inline playbooks/tasks/portal-role-task.yml:371]: Why is this here and not part of the abuse scanner?

Comment (peterjan) [reply playbooks/tasks/portal-role-task.yml:371]: `cypress` image is ran from the container on the host itself, so it is preferred to have the cypress image be available on all servers instead of pulling it in on the fly

File: playbooks/templates/.env.j2 (hunk 1 above)
Comment (peterjan) [reply playbooks/templates/.env.j2:299]: I can throw this out. I thought we usually add an optional variable for every environment variable we use from the environment so it's configurable should portal operators want to deviate from the default value.

**Review by MSevey — APPROVED** [review]

**Review by firyx — APPROVED** [review]
